### PR TITLE
OBSDOCS-219 - Child PR - Loki GUI Install.

### PR DESCRIPTION
**Pull request by libander**
Child PR of https://github.com/openshift/openshift-docs/pull/59285 for merge review ease. 
All approvals completed on parent PR. 

Preview: https://65234--docspreview.netlify.app/openshift-enterprise/latest/logging/cluster-logging-loki#logging-loki-gui-install_cluster-logging-loki

### CODE REVIEW

File: modules/logging-loki-gui-install.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+{Product-title} Operators use custom resources (CR) to manage applications and their components. High-level configuration and settings are provided by the user within a CR. The Operator translates high-level directives into low-level actions, based on best practices embedded within the Operator’s logic. A custom resource definition (CRD) defines a CR and lists all the configurations available to users of the Operator. Installing an Operator creates the CRDs, which are then used to generate CRs.
+
+.Prerequisites
+
+* Supported object store (AWS S3, Google Cloud Storage, Azure, Swift, Minio, OpenShift Data Foundation)
+
+.Procedure
+
+. Install the `Loki Operator`:
```
Comment (abrennan89) [inline modules/logging-loki-gui-install.adoc:17]:
```suggestion
. Install the Loki Operator:
```
Operator names shouldn't be in backticks

File: modules/logging-loki-gui-install.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+[NOTE]
+====
+If the `openshift-logging` namespace already exists, you must add the namespace label, `openshift.io/cluster-monitoring: "true"`, to enable metrics service discovery.
+====
+
+.. Under *Update approval* select *Automatic*.
++
+--
+include::snippets/logging-approval-strategy-snip.adoc[]
+--
```
Comment (abrennan89) [inline modules/logging-loki-gui-install.adoc:73]: If you're using snippets, the metadata for the snippets must be updated, i.e. the list of places where it's used in the snippet file itself: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#text-snippet-file-metadata

Please make sure that metadata is updated for all modules and snippets

File: modules/logging-loki-gui-install.adoc [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object. You must select this option to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
+
+.. For *Update approval* select *Automatic*, then click *Install*.
++
+--
+include::snippets/logging-approval-strategy-snip.adoc[]
+--
+
+.Procedure
```
Comment (abrennan89) [inline modules/logging-loki-gui-install.adoc:46]: There should only be one procedure per module. Per the mod docs guidelines, "A procedure module contains numbered, step-by-step instructions to help the user accomplish a single task."

**Comment by libander** [comment]
@lpettyjo - manual CP opened.

**Pull request merged**